test/common.py

```diff
@@ -863,6 +863,7 @@ def setUp(self):
     super().setUp()
     self.js_engines = config.JS_ENGINES.copy()
     self.settings_mods = {}
+    self.skipExec = None
```

Comment:
```suggestion
    self.skip_exec = None
```
That would match the code around it. Or is this intentionally more similar to `skipTest` from the standard library?

Reply:
Done